css/style.css

```diff
@@ -116,10 +145,246 @@ body {
 }
 
 .my-recent-works {
-  padding-top: 5px;
+  padding-top: 10px;
   text-align: center;
   padding-left: 2%;
   padding-right: 2%;
-  padding-bottom: 30px;
-  line-height: 2rem;
 }
+
+.my-recent-works h3 {
+  line-height: 3rem;
+  font-weight: 700;
+  font-size: 40px;
+}
+
+.my-recent-works img {
+  margin-top: 20px;
+}
+
+/* ---------------------mobile version------------ */
+
+.my-works h1 {
+  margin-bottom: 20px;
+  margin-top: 50px;
+  text-align: center;
+  font-size: 30px;
+  font-weight: bold;
+  line-height: 1.2em;
+}
+
+.my-works {
+  max-width: 375px;
+  text-align: center;
+}
+
+.my-works .indicator {
+  margin-bottom: 10px;
+  text-align: center;
+}
+
+.works {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 20px;
+  padding: 10px;
+}
+
+.work-1,
+.work-2,
+.work-3,
+.work-4,
+.work-5,
+.work-6 {
```

Comment:
- [ ] You did an amazing job using diplay:grid for laying out the elements inside of the cards but as it was part of the [project requiremtens](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) you have to use display:grid for the main container of projects and Flexbox for individual cards. Kindly change that 💪